docs/getting-started/full-site-editing.md

```diff
@@ -27,12 +27,9 @@ An ongoing [FSE Outreach program](https://make.wordpress.org/test/handbook/full-
 
 ## Block Themes
 
-If you are using the Gutenberg plugin you can run, test, and develop block themes. Block themes are themes built using templates composed using blocks. See [block theme overview](/docs/how-to-guides/themes/block-theme-overview.md) for additional details.
-
-- See the [Create a Block Theme](/docs/how-to-guides/themes/create-block-theme.md) tutorial for a walk-through of the pieces of a block theme.
+If you are using the Gutenberg plugin you can run, test, and develop block themes. Block themes are themes built using templates composed using blocks. See the [block theme documentation](https://developer.wordpress.org/themes/block-themes/) for additional details.
```

Comment:
I know this is off topic because all that is changed is the link, but I don't think this is accurate anymore 😃  we don't need Gutenberg to be active to run, test and develop block themes.

Reply:
Haha yup, I will remove that first line as well. I was just focused on the link. There is a TON of this outdated info in the docs.